### PR TITLE
OZ-75: Cleaning up metadata and loosely align it for Reference Demo Data.

### PR DESCRIPTION
**Pull request by Ruhanga**

### CODE REVIEW

File: distro/configuration/relationshiptypes/relationshiptypes.csv [hunk 1]
```diff
@@ -1,7 +1,4 @@
 Uuid,Void/Retire,Name,Description,Weight,Preferred,A is to b,B is to a
-2026807f-f0f1-4d87-892d-2abaa53358ae,,Spouse/Spouse,,A relationship from a spouse to their spouse,,,Spouse,Spouse
-048bd9e2-072a-46f6-b720-7e6bf4a6f1fe,,Parent/Child,A relationship from a mother/father to the child,,,Parent,Child
-c0165e10-f147-4840-a65e-e234a674706a,,Sibling/Sibling,A relationship between brother/sister or brother/brother or sister/sister,,,Sibling,Sibling
```
[the remark this answers is not in the document]
Comment (Ruhanga) [reply distro/configuration/relationshiptypes/relationshiptypes.csv:4]: These already exist @ibacher,  see [this](https://github.com/openmrs/openmrs-core/blob/b7c03a28c64493023a8211c53bb60d4d944b39b7/liquibase/src/test/resources/org/openmrs/liquibase/snapshots/core-data/liquibase-core-data-UPDATED-SNAPSHOT.xml#L4999-L5068).

File: distro/configuration/concepts/programs_workflows_states_core-demo.csv [hunk 2]
```diff
@@ -1,7 +1,5 @@
 Uuid,Void/Retire,Same as mappings,Fully specified name:en,Fully specified name:fr,Fully specified name:es,Short name:en,Short name:fr,Short name:es,Description:en,Description:fr,Description:es,Data class,Data type,_version:1,_order:1000
-7dc37500-1725-11ed-861d-0242ac120002,,SNOMED MVP:HIVD,Human immunodeficiency virus (HIV) disease,,,HIV disease,,,Human immunodeficiency virus (HIV) disease program,,,Misc,N/A,,
 7dc376cc-1725-11ed-861d-0242ac120002,,SNOMED MVP:PMTCT,Prevention of Maternal to Child HIV transmissio,,,PMTCT,,,Prevention of Maternal to Child HIV transmission program,,,Misc,N/A,,
-7dc37906-1725-11ed-861d-0242ac120002,,SNOMED MVP:HIV_PEP,Pre-exposure prophylaxis (PrEP),,,Pre-Exposure Prophylaxis HIV Prevention,,,Prevention of HIV through Pre-Exposure Prophylaxis program,,,Misc,N/A,,
```
[the remark this answers is not in the document]
Comment (Ruhanga) [reply distro/configuration/concepts/programs_workflows_states_core-demo.csv:4]: Yes we do and these come through some ocl package as can be seen on a running instance.

File: distro/configuration/relationshiptypes/relationshiptypes.csv (hunk 1 above)
[the remark this answers is not in the document]
Comment (ibacher) [reply distro/configuration/relationshiptypes/relationshiptypes.csv:4]: Shouldn't we also get rid of the Uncle/Nephew Aunt/Niece pairs too?

**Review by ibacher — APPROVED** [review]

File: distro/configuration/relationshiptypes/relationshiptypes.csv (hunk 1 above)
[the remark this answers is not in the document]
Comment (Ruhanga) [reply distro/configuration/relationshiptypes/relationshiptypes.csv:4]: The relationship in the changeset is more generic and should look like _Aunt/Uncle_ **/** _Niece/Nephew_ type which is not being violated/duplicated by the more specific entries in this `csv`.

**Comment by Ruhanga** [comment]
I've commented about the same above. Thanks for the review @ibacher.

**Pull request merged**